test/skipped/advi_demo.jl

```diff
@@ -0,0 +1,129 @@
+using Turing
+using Turing: Variational
+
+
+@model model(x) = begin
+    s ~ InverseGamma(2,3)
+    m ~ Normal(0.0, sqrt(s))  # `Normal(μ, σ)` has mean μ and variance σ², i.e. parametrize with std. not variance
+    for i = 1:length(x)
+        x[i] ~ Normal(m, sqrt(s))
+    end
+end
+
+# generate data
+x = randn(1, 2000);
+
+# construct model
+m = model(x)
+
+# ADVI
+using Flux.Optimise
+
+opt = ADAGrad()                  # optimizer taking value and gradient to new value
+advi = ADVI(10, 100)             # <: VariationalInference
+q = vi(m, advi; optimizer = opt) # => MeanField <: VariationalPosterior
+
+elbo = Variational.ELBO()        # <: VariationalObjective
+
+history = [elbo(q, m, 1000)]     # history of objective evaluations
+
+# setup for plotting
+using Plots, StatsPlots, LaTeXStrings
```

Comment:
If possible a unit-test should not require plotting.

Reply:
100% agree; see previous comment regarding this file.